blackbox/docs/enterprise/monitoring.txt

```diff
@@ -0,0 +1,98 @@
+.. _jmx_monitoring_plugin:
+
+=====================
+JMX Monitoring Plugin
+=====================
+
+The CrateDB monitoring plugin exposes query metrics via the `JMX`_ API.
+
+This plugin is an `enterprise`_ feature.
```

Comment:
if this is the phrasing we're going with, sure! I'll update some of the other mentions to use "Enterprise Edition" (presuming capitalised like this)